modelci/hub/converter/to_tvm.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#  Copyright (c) NTU_CAP 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+import tvm
+import numpy 
+import torch
+import torch.jit
+from torchvision import transforms
+
+from tvm import relay
+
+from pathlib import Path
+from PIL import Image
+
+import sys
+sys.path.append('../')
```

Comment:
You do not need to declare the system path here if you have added the Python path to the project workspace.